std/regex/package.d

```diff
@@ -1248,18 +1235,21 @@ public @trusted void replaceAllInto(alias fun, Sink, R, RegEx)
     replaceMatchesInto!fun(sink, input, matchAll(input, re));
 }
 
-// a bit of examples
+///
 @system unittest
 {
-    //swap all 3 letter words and bring it back
-    string text = "How are you doing?";
-    auto sink = appender!(char[])();
-    replaceAllInto!(cap => retro(cap[0]))(sink, text, regex(`\b\w{3}\b`));
-    auto swapped = sink.data.dup; // make a copy explicitly
-    assert(swapped == "woH era uoy doing?");
-    sink.clear();
-    replaceAllInto!(cap => retro(cap[0]))(sink, swapped, regex(`\b\w{3}\b`));
-    assert(sink.data == text);
+    // insert comma as thousands delimiter in fifty randomly produced big numbers
+    import std.array, std.random, std.conv, std.range;
+    static re = ctRegex!(`(?<=\d)(?=(\d\d\d)+\b)`, "g");
```

Comment:
It's just a habit I developed since I heard about the runtime benefits. Seeing as the main reason I want to switch over to D from Python is the speed issue, I wanted to develop the "fastest" habits. If there is a problem in using it in the unittest I have no objection to changing it.